.gitignore

```diff
@@ -36,8 +36,8 @@ bower_components
 typings
 
 # transpiled typescript
-infiniboard-app/app/*.js
-infiniboard-app/app/*.js.map
+infiniboard-app/**/*.js

```

Comment:
Should not be necessary, as typescript gets compiled to `build/app` now.
This expression also matches `node_modules/**/*.js` which is not intended.

Just revert change.